atomic_reactor/plugins/exit_koji_promote.py

```diff
@@ -574,26 +575,11 @@ def login(self):
         :return: koji.ClientSession instance, logged in
         """
         session = koji.ClientSession(self.kojihub)
```

Comment:
Good point. Maybe a `create_session(bool:login)` method that creates the session and conditionally performs login?